### PR TITLE
Storefront uses Flyout for Mobile Menu

### PR DESCRIPTION
**Pull request by stephanschroeter**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/housekeeping/mobile-menu-storefront

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves [IssueNumber]

### CODE REVIEW

**Review by tatjana-gartner2 — APPROVED** [review]
Looks good to me :)

**Comment by o-hook** [comment]
I don't know if this is a general bug in the `flyout` component or the special integration in the Storefront: I can access the flyout menu items (links, accordion) with the keyboard by tabbing even if the `flyout` is hidden.

**Comment by stephanschroeter** [comment]
> I don't know if this is a general bug in the `flyout` component or the special integration in the Storefront: I can access the flyout menu items (links inside open accordion) with the keyboard by tabbing even if the `flyout` is hidden.

It's a bug when flyout is used in combination with accordion. I'll create a bug issue.

File: packages/components/src/components/flyout/flyout.examples.md [hunk 1]
```diff
@@ -88,12 +88,13 @@ content. The following example shows the visualization of the grid if used insid
 
 ## Custom styling
 
-The `p-flyout` component has a default of `max-width: 1180px`. This value can be overwritten by CSS Custom Properties
-(aka CSS Variables):
+The `p-flyout` component has a default of `max-width: 1180px` and `width: auto`. This values can be overwritten by CSS
```
Comment (henrijoss) [inline packages/components/src/components/flyout/flyout.examples.md:91]:
```suggestion
The `p-flyout` component has a default of `max-width: 1180px` and `width: auto`. These values can be overwritten by CSS
```

**Review by henrijoss — APPROVED** [review]

**Pull request merged**